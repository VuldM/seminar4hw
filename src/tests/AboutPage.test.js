import {render, screen} from "@testing-library/react";
import AboutPage from "../components/AboutPage";
import React from "react";


test("render about page", () => {
    render(<AboutPage />);
    expect(screen.getByText("About Us")).toBeInTheDocument();
});

test("about us snapshot test", () => {
    const {container} = render(<AboutPage/>);
    expect(container).toMatchSnapshot();
});