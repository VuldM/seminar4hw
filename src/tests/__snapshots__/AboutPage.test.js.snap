// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`about us snapshot test 1`] = `
<div>
  <h1>
    About Us
  </h1>
</div>
`;
